documentation/book/kafka-exporter/proc-kafka-exporter-configuring.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-exporter-configuration.adoc
+
+[id='proc-kafka-exporter-configuring-{context}']
+= Configuring Kafka Exporter
+
+Configure the Kafka Exporter in the `Kafka` resource through `KafkaExporter` properties.
+
+Refer to the xref:ref-sample-kafka-resource-config-deployment-configuration-kafka[sample Kafka YAML configuration] for an overview of the `Kafka` resource and its properties.
+
+The properties relevant to the Kafka Exporter configuration are shown in this procedure.
+
+You can configure these properties as part of a deployment or redeployment of the Kafka cluster.
+
+.Prerequisites
+
+* A Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `KafkaExporter` properties for the `Kafka` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+KafkaExporter:
+  image: my-org/my-image:latest <1>
+  groupRegex: ".*" <2>
+  topicRegex: ".*" <3>
+  resources: <4>
+    requests:
+      cpu: 200m
+      memory: 64Mi
+    limits:
+      cpu: 500m
+      memory: 128Mi
+  logging: <5>
```

Comment:
The logging here is different. It is just this:

```yaml
logging: debug
```

Valid values are `debug`, `info`, `warn`, `error`, `fatal`.

Reply:
Okay. I've removed the link too, as we don't need it.